src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveFiles/ExpectedOutputs/SEC103_002.ASCIIArmoredFile_fake.sarif

```diff
@@ -74,13 +74,13 @@
             }
           ],
           "fingerprints": {
-            "secretHashSha256/v0": "cd2099a31af74edb52f9e10e620289f310c43b5f3dcdb41ad30b8a0b2a4e14c6",
+            "secretHashSha256/v0": "731584c0c360c3be368abcdb3a5cce5fb3261b432a9f6c306a5688d79016b37e",
```

Comment:
>731584c0c360c3be368abcdb3a5cce5fb3261b432a9f6c306a5688d79016b37e [](http://example.com/codeflow?start=36&length=64)

The input file has the path with `..\Src\..` and I changed it to `..\src\..` so the hash is different.

Reply:
Nice note, thanks for the explanation.